### PR TITLE
Updated Index.html

### PR DESCRIPTION
**Pull request by MuretiMartin**
Updated Index.html

### CODE REVIEW

**Comment by MuretiMartin** [comment]
Please revert the changes

File: index.html [hunk 1]
```diff
@@ -4,18 +4,18 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Git & GitHub Crash Course</title>
+    <title>Git & GitHub Crash Course Hello</title>
```
Comment (v-muretimartin) [inline index.html:7]: revert the changes

File: index.html [hunk 2]
```diff
@@ -4,18 +4,18 @@
...
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
     <header>
       <img id="logo" src="logo.png" alt="A logo that contains the word 'Git'" />
-      <h1>Git & GitHub Crash Commit course </h1>
+      <h1>Git & GitHub Crash Commit course - Hello </h1>
```
Comment (v-muretimartin) [inline index.html:13]: revert the changes

File: index.html [hunk 3]
```diff
@@ -4,18 +4,18 @@
...
     </header>
     <main>
       <p>Because the basics are important! We start here</p>
       <p>
-        Learn more about Git in the
+        Learn more about Git in the -- Hello
```
Comment (v-muretimartin) [inline index.html:18]: revert the changes
